hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/meta/QueryContext.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.meta;
+
+import com.google.protobuf.ByteString;
+
+/**
+ * Provides context for query processing.
+ */
+public interface QueryContext {
+    ByteString getLedgerId();
```

Comment:
This adds protobuff as a dependency to the api layer. Maybe we can create a `LedgerId` interface that has a `getBytes()` method?

Reply:
Agreed. Providing it as a String